x/oracle/client/common/proof/multi_store.go

```diff
@@ -10,59 +10,59 @@ import (
 // compute (in combination with oracle store hash) Tendermint's application state hash at a given block.
 //                                              ________________[AppHash]_________________
 //                                             /                                          \
-//                         _________________[I14]_________________                        [G]
-//                        /                                        \
-//             _______[I12]______                          _______[I13]________
+//                         _________________[I14]_________________                     __[I15]__
+//                        /                                        \				 /          \
+//             _______[I12]______                          _______[I13]________    [G]         [H]
 //            /                  \                        /                    \
 //       __[I8]__             __[I9]__                __[I10]__              __[I11]__
 //      /         \          /         \            /          \            /         \
 //    [I0]       [I1]     [I2]        [I3]        [I4]        [I5]        [I6]       [I7]
 //   /   \      /   \    /    \      /    \      /    \      /    \      /    \     /    \
 // [0]   [1]  [2]   [3] [4]   [5]  [6]    [7]  [8]    [9]  [A]    [B]  [C]    [D]  [E]   [F]
-// [0] - auth     [1] - authz    [2] - bank    [3] - capability [4] - crisis  [5] - dist
-// [6] - evidence [7] - feegrant [8] - gov     [9] - ibccore    [A] - mint    [B] - oracle
-// [C] - params   [D] - slashing [E] - staking [F] - transfer   [G] - upgrade
+// [0] - auth     [1] - authz    [2] - bank     [3] - capability [4] - crisis   [5] - dist
```

Comment:
<img width="525" alt="image" src="https://user-images.githubusercontent.com/13800683/185667717-b8a52639-cd4a-46ec-aecc-14dedb3a537e.png">

@taobun @satawatnack Are these names StoreKey? If yes, should we write "acc (auth)" instead to show actual sorted names? just in case we have "a[c-u]" module in the future.